tests/scripts/thread-cert/border_router/MATN_02_MLRFirstUse.py

```diff
@@ -0,0 +1,308 @@
+
+        # Ensure the topology is formed correctly
+        pv.verify_attached('TD', 'BR_1')
+        pv.verify_attached('BR_2')
+
+        # 1. TD Registers for multicast address, MA1, at BR_1.
+        # TD unicasts an MLR.req CoAP request to BR_1 as
+        # "coap://[<BR_1 RLOC or PBBR ALOC>]:MM/n/mr".
+        # The payload contains "IPv6Address TLV: MA1".
+        pkts.filter_wpan_src64(vars['TD']) \
+            .filter_ipv6_2dsts(vars['BR_1_RLOC'], vars['LEADER_ALOC']) \
+            .filter_coap_request('/n/mr') \
+            .filter(lambda p: p.thread_meshcop.tlv.ipv6_addr == [MA1]) \
+            .must_next()
+
+        # 3. BR_1 responds to the multicast registration.
+        # BR_1 unicasts an MLR.rsp CoAP response to TD as "2.04 changed".
+        # The payload contains "Status TLV: ST_MLR_SUCCESS".
+        pkts.filter_wpan_src64(vars['BR_1']) \
+            .filter_ipv6_dst(vars['TD_RLOC']) \
+            .filter_coap_ack('/n/mr') \
+            .filter(lambda p: p.thread_nm.tlv.status == 0) \
+            .must_next()
+
+        # 3a. BR_2 does not respond to the multicast registration.
+        pkts.filter_wpan_src64(vars['BR_2']) \
+            .filter_ipv6_dst(vars['TD_RLOC']) \
+            .filter_coap_ack('/n/mr') \
+            .must_not_next()
+
+        # 4. BR_1 informs other BBRs on the network of multicast registration.
+        # BR_1 multicasts a BMLR.ntf CoAP request to the Backbone Link including
+        # to BR_2, as follows
+        pkts.filter_eth_src(vars['BR_1_ETH']) \
+            .filter_ipv6_dst(config.ALL_NETWORK_BBRS_ADDRESS) \
+            .filter_coap_request('/b/bmr') \
+            .filter(lambda
+                        p: p.thread_meshcop.tlv.ipv6_addr == [MA1] and
+                           p.thread_bl.tlv.timeout == MLR_NOTIFICATION_TIMEOUT) \
+            .must_next()
+
+        # 6. BR_1 multicasts an MLDv2 message to start listening to MA1.
+        # BR_1 multicasts an MLDv2 message of type “Version 2 Multicast Listener
+        # Report” (see [RFC 3810] Section 5.2). Where:
+        # Nr of Mcast Address at least 1 Records (M): Multicast Address Record
+        # The Multicast Address Record contains the following:
+        # Record Type: 4 (CHANGE_TO_EXCLUDE_MODE)
+        # Number of Sources (N): 0
+        # Multicast Address: MA1
+        # TODO: Implement this verification
+
+        # 7. Host sends a ping packet to the multicast address, MA1.
+        _pkt = pkts.filter_eth_src(vars['Host_ETH']) \
+            .filter_ipv6_dst(MA1) \
+            .filter_ping_request() \
+            .must_next()
+
+        # 8. BR_2 does not forward the ping packet with multicast address MA1 to
+        # its Thread Network.
+        pkts.filter_wpan_src64(vars['BR_2']) \
+            .filter_AMPLFMA() \
+            .filter_ping_request(identifier=_pkt.icmpv6.echo.identifier) \
+            .filter(check_mpl_seed_id(vars['BR_2_RLOC'], vars['BR_2_RLOC16'])) \
```

Comment:
```suggestion
            .filter_AMPLFMA(check_mpl_seed_id=(vars['BR_2_RLOC'], vars['BR_2_RLOC16'])) \
            .filter_ping_request(identifier=_pkt.icmpv6.echo.identifier) \
```
Add this common logic to `filter_AMPLFMA` ?